### DISCUSSION TITLE
new method to completely remove edge warping

### DISCUSSION THREAD

**Discussion by loawizard** [nagadomi/nunif#113] [Ideas]
https://youtu.be/YYFUXv-uF7g?si=OdMagMdSQqjJShFh

These guys figured out how to completely remove edge warping.

"Authors: Lukas Mehl; Andrés Bruhn; Markus Gross; Christopher Schroers
 Description: Despite of exciting advances in image-based rendering and novel view synthesis, it is still challenging to achieve high-resolution results that can reach production-level quality when applying such methods to the task of stereo conversion. At the same time, only very few dedicated stereo conversion approaches exist, which also fall short in terms of the required quality. Hence, in this paper, we present a novel method for high-resolution 2D-to-3D conversion. It is fully differentiable in all of its stages and performs disparity-informed warping, consistent foreground-background compositing, and background-aware inpainting. To enable temporal consistency in the resulting video, we propose a strategy to integrate information from additional video frames. Extensive ablation studies validate our design choices, leading to a fully automatic model that outperforms existing approaches by a large margin (49-70% LPIPS error reduction). Finally, inspired from current practices in manual stereo conversion, we introduce optional interactive tools into our model, which allow to steer the conversion process and make it significantly more applicable for 3D film production. "
![image](https://github.com/nagadomi/nunif/assets/153664373/f5f9a0b4-0697-4790-aa25-aa4f0a2efd48)

**Comment by kaelsonofkrypto** [comment]
Yessss please investigate this if possible, this would totally help

**Comment by donrikk** [comment]
oh yeah ide love to see if this is implementable!

**Comment by beosliege** [comment]
Looks really promising! But I don't see a github project or other source by which to leverage this easily right now. Am I missing something?

**Comment by nagadomi** [comment]
At the moment I have no plans to implement inpainting.
Perhaps I will try in the future, but right now I am trying to avoid using it.

**Reply by tufeixp** [reply]
> At the moment I have no plans to implement inpainting. Perhaps I will try in the future, but right now I am trying to avoid using it.

[StereoCrafter](https://huggingface.co/TencentARC/StereoCrafter) inpainting is out and the result is good, better impainting will bring more depth and also techs like text fragmentation and timeline system build-in should also be used if you wish to bring iw3 to a professional stereorizer tool. GOD I wish I can invest money on your project!
